src/panels/config/js/script/call_service.tsx

```diff
@@ -39,6 +83,16 @@ export default class CallServiceAction extends AutomationComponent<any> {
           value={service}
           onChange={this.serviceChanged}
         />
+        {entity && (
+          <ha-entity-picker
+            hass={hass}
+            value={entity_id}
+            label={entity.description}
+            onChange={this.entityChanged}
+            include-domains={JSON.stringify([computeDomain(service)])}
```

Comment:
Shouldn't this work too? `includeDomains={[computeDomain[service]]}`

Reply:
Preact removes the array?

Reply:
Preact checks `prop in elem`, and sets the value as a JS object instead of setting it as an attribute.